workspace-manager-clienttests/src/main/resources/servers/workspace-dev.json

```diff
@@ -4,7 +4,7 @@
 
   "samUri": "https://sam.dsde-dev.broadinstitute.org",
   "datarepoUri": "https://jade.datarepo-dev.broadinstitute.org/",
-  "workspaceManagerUri": "https://workspace.dsde-dev.broadinstitute.org",
+  "workspaceManagerUri": "http://localhost:8080",
```

Comment:
I think WSM dev should not point to localhost. We can make a new server config, maybe called workspace-localhost.json?

Reply:
A local server config has been added for integration tests.